src/commands/ban.ts

```diff
@@ -163,30 +172,38 @@ const command = new SlashCommandBuilder()
 	.setDefaultMemberPermissions('0')
 	.setName('ban')
 	.setDescription('Ban user(s)')
-	.addStringOption(option => {
-		return option.setName('users')
-			.setDescription('User(s) to ban')
-			.setRequired(true);
-	})
-	.addStringOption(option => {
-		return option.setName('reason')
-			.setDescription('Reason for the ban')
-			.setRequired(true);
-	})
-	.addNumberOption(option => {
-		return option.setName('delete_messages')
-			.setDescription('How much of their recent message history to delete')
-			.addChoices(
-				{ name: 'Previous 30 Minutes', value: 30 * 60 },
-				{ name: 'Previous Hour', value: 60 * 60 },
-				{ name: 'Previous 3 Hours', value: 3 * 60 * 60 },
-				{ name: 'Previous 6 Hours', value: 6 * 60 * 60 },
-				{ name: 'Previous 12 Hours', value: 12 * 60 * 60 },
-				{ name: 'Previous Day', value: 24 * 60 * 60 },
-				{ name: 'Previous 3 Days', value: 3 * 24 * 60 * 60 },
-				{ name: 'Previous Week', value: 7 * 24 * 60 * 60 },
-			);
-	});
```

Comment:
Whats up with the git history here? Is this based on another branch?

Reply:
Yes, all of these things are because this PR is based on #39.

Reply:
You're entirely correct, that's my fault. I did not see the base branch and thought this was based on dev, that's my bad